src/art/Anurag/script.js

```diff
@@ -0,0 +1,54 @@
+    //Grab the canvas element on the client 
+    // and get the context to add stuff to the canvas
+    var canvas = document.getElementById("myCanvas");
+    var CanvasContext = canvas.getContext("2d");
```

Comment:
Nitpicking: In JavaScript, your variables shouldn't be uppercase, unless you plan to create an object that you want to instantiate with the `new` keyword. That's because JavaScript does some weird things with object prototypes.

camelCase is the most appropriate way, so `canvasContext`.